model/vcs/metrics.yaml

```diff
@@ -0,0 +1,147 @@
+groups:
+  - id: metric.vcs.change.count
+    type: metric
+    metric_name: vcs.change.count
+    brief: 'The number of changes (pull requests/merge requests/changelists) in a repository, categorized by their state (e.g. open or merged)'
+    instrument: updowncounter
```

Comment:
Did we want to reset this back to it's original fundamental type of gauge?

Reply:
No, because we can aggregate away the different attributes giving for example the sum of open PRs for all tracked repos or total changes for a given repo.
For that updowncounter is the correct instrument.